src/app/auth/account/account.component.ts

```diff
@@ -1,8 +1,190 @@
-import { ChangeDetectionStrategy, Component } from "@angular/core";
+import { ModalService } from "./../../components/modal/modal.service";
+import {
+    DatasetsByAccountNameQuery,
+    DatasetSearchOverviewFragment,
+    PageBasedInfo,
+    User,
+} from "./../../api/kamu.graphql.interface";
+import { BaseComponent } from "src/app/common/base.component";
+import { NavigationService } from "src/app/services/navigation.service";
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    OnInit,
+    ViewChild,
+} from "@angular/core";
+import { AuthApi } from "src/app/api/auth.api";
+import { AccountDetailsFragment } from "src/app/api/kamu.graphql.interface";
+import { MaybeNull } from "src/app/common/app.types";
+import { AccountTabs } from "./account.constants";
+import {
+    ActivatedRoute,
+    NavigationEnd,
+    Params,
+    Router,
+    RouterEvent,
+} from "@angular/router";
+import { filter, map } from "rxjs/operators";
+import AppValues from "src/app/common/app.values";
+import { promiseWithCatch } from "src/app/common/app.helpers";
 
 @Component({
     selector: "app-account",
     templateUrl: "./account.component.html",
+    styleUrls: ["./account.component.sass"],
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class AccountComponent {}
+export class AccountComponent extends BaseComponent implements OnInit {
+    public accountViewType = AccountTabs.overview;
+    private _window: Window;
+    private resizeId: NodeJS.Timer;
+    private menuRowWidth: number;
+    public user: MaybeNull<AccountDetailsFragment>;
+    public datasets: DatasetSearchOverviewFragment[] = [];
+    public pageInfo: PageBasedInfo;
+    public accountTabs = AccountTabs;
+    public accountName: string;
+    public isClickableRow = true;
+    public isDropdownMenu = false;
+    public isCurrentUser = false;
+    public currentPage = 1;
+
+    @ViewChild("containerMenu") containerMenu: ElementRef;
+    @ViewChild("dropdownMenu") dropdownMenu: ElementRef;
+
+    constructor(
+        private authApi: AuthApi,
+        private route: ActivatedRoute,
+        private router: Router,
+        private navigationService: NavigationService,
+        private cdr: ChangeDetectorRef,
+        private modalService: ModalService,
+    ) {
+        super();
+        this._window = window;
+        this.user = this.authApi.currentUser;
```

Comment:
I think the user should come from the URL, not from the login.  We might be viewing someone else's account, not necessarily the account of the currently logged in user.